Tiltfile

```diff
@@ -183,9 +202,12 @@ k8s_resource(
 k8s_resource(
   new_name = 'crds',
   objects = [
+    'clusterpromotiontasks.kargo.akuity.io:customresourcedefinition',
     'freights.kargo.akuity.io:customresourcedefinition',
+    'projectconfigs.kargo.akuity.io:customresourcedefinition',
     'projects.kargo.akuity.io:customresourcedefinition',
     'promotions.kargo.akuity.io:customresourcedefinition',
+    'promotiontasks.kargo.akuity.io:customresourcedefinition',
```

Comment:
This is incidental cleanup that seemed ok to tackle while already touching this file.